src/main.rs

```diff
@@ -260,10 +254,11 @@ pub fn main() {
         } else if matches.opt_present("dry-run") {
             dry_run(&input, opts);
         } else {
-            let p = Parser::new_ext(&input, opts);
+            let mut p = Parser::new_ext(&input, opts);
             let stdio = io::stdout();
             let buffer = std::io::BufWriter::with_capacity(1*1024*1024, stdio.lock());
-            html::write_html(buffer, p).unwrap();
+            html::write_html(buffer, &mut p).unwrap();
+            mem::forget(p);
```

Comment:
I'd add a comment here why we're doing this, because it's unexpected. People copying the code *should* do it if it's a simple commandline, but *should not* if it's processing lots of content or remaining online.

Reply:
Good point, will do!